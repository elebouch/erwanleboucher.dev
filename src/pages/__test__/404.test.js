import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import NotFound from "../404"

describe("NotFound", () => {
  it("renders correctly", () => {
    const { container } = render(<NotFound />)
    expect(container.innerHTML).toMatchSnapshot()
  })
})
