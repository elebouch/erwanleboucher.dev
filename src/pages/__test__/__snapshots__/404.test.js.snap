// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `"<article><h1>NOT FOUND</h1><p>You just hit a route that doesn't exist... the sadness.</p></article>"`;
